src/content/extensions/multipleroutes/src/options.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+// Saves options.
+function saveOptions() {
+  var multiRoutes;
+  var nonProxiedUdp;
+  var radios = document.getElementsByName('routeselection');
+  for (var i = 0, length = radios.length; i < length; i++) {
+    if (radios[i].checked) {
+      multiRoutes = i < 1;
+      nonProxiedUdp = i < 2;
+      break;
+    }
+  }
+
+  chrome.privacy.network.webRTCMultipleRoutesEnabled.set({
+    'value': multiRoutes
+  });
+  try {
+    chrome.privacy.network.webRTCNonProxiedUdpEnabled.set({
+      'value': nonProxiedUdp
+    });
+  } catch (err) {
+    // Shouldn't run into here.
+  }
+}
+
+function restoreRadios(multiRoutes, nonProxiedUdp) {
+  var radios = document.getElementsByName('routeselection');
+  if (multiRoutes) {
+    if (nonProxiedUdp) {
+      radios[0].checked = true;
+    } else {
+      radios[0].checked = false;
+      alert(
+        '{multiRoutes: true, nonProxiedUdp: false} is not a supported option.'
+      );
+    }
+  } else {
+    if (nonProxiedUdp) {
+      radios[1].checked = true;
+    } else {
+      radios[2].checked = true;
+    }
+  }
+}
+
+// Restores checkbox states.
+function restoreOptions() {
+  var multiRoutes = true;
+  var nonProxiedUdp = true;
+  chrome.privacy.network.webRTCMultipleRoutesEnabled.get({},
+    function(details) {
+      multiRoutes = details.value;
+      restoreRadios(multiRoutes, nonProxiedUdp);
+    });
+  try {
+    chrome.privacy.network.webRTCNonProxiedUdpEnabled.get({},
+      function(details) {
+        nonProxiedUdp = details.value;
+        restoreRadios(multiRoutes, nonProxiedUdp);
+      });
+    document.getElementById('for_multirouteOffUdpOff_notSupported').innerHTML =
+      '';
+  } catch (err) {
```

Comment:
we have the minimum chrome version specified so multipleRoutes is always supported.